lib/hammer_cli_foreman/registration_tokens.rb

```diff
@@ -0,0 +1,25 @@
+module HammerCLIForeman
+  class RegistrationTokens < HammerCLIForeman::Command
+    resource :registration_tokens
+    command_name 'registration-tokens'
+    desc _('Manage registration tokens')
+
+    class InvalidateMultipleCommand < HammerCLIForeman::DeleteCommand
+      action :invalidate_jwt_tokens
+      command_name 'invalidate-multiple'
+      success_message _('Successfully invalidated registration tokens for %{users}.')
+
+      build_options
+    end
+
+    class InvalidateCommand < HammerCLIForeman::DeleteCommand
+      action :invalidate_jwt
+      command_name 'invalidate'
+      success_message _('Successfully invalidated registration tokens for %{user}.')
```

Comment:
DEPENDS ON [REF](https://github.com/theforeman/foreman/pull/10412/commits/d91068ef146469907a37e5051aa447739d00a280#diff-d61ad6061120365f1c156b6a821377f022e3e9a223561d692c1961a62637f845R38)
We need to merge that before we merge this